### PR TITLE
fix: enable ssl as default, using placeholder certificate

### LINKED ISSUE #3005: feat: enable the SSL feature in DP by default
APISIX version: `2.1`

It not a good idea to disable the SSL feature by default, and it is not easy to enable this feature.
we should enable it by default.

Then we can use the placeholder certificate, and then APISIX can start normally. 

image

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:
Fix #3005 

We still need to enable SSL as default, thus using Random certificate as the placeholder

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: conf/config-default.yaml [hunk 1]
```diff
@@ -64,8 +64,8 @@ apisix:
   # https_admin: true             # enable HTTPS when use a separate port for Admin API.
                                 # Admin API will use conf/apisix_admin_api.crt and conf/apisix_admin_api.key as certificate.
   admin_api_mtls:               # Depends on `port_admin` and `https_admin`.
-    admin_ssl_cert: ""             # Path of your self-signed server side cert.
-    admin_ssl_cert_key: ""         # Path of your self-signed server side key.
+    admin_ssl_cert: "cert/mtls_PLACE_HOLDER.crt"             # Path of your self-signed server side cert.
```
Comment (spacewander) [inline conf/config-default.yaml:67]: The certificate for admin can't be a placeholder.

**Review by spacewander — CHANGES_REQUESTED** [review]

File: conf/config-default.yaml (hunk 1 above)
Comment (Yiyiyimu) [reply conf/config-default.yaml]: already removed. thx!

File: conf/config-default.yaml [hunk 2]
```diff
@@ -101,15 +101,14 @@ apisix:
...
-                                  # enable it to use your own cert and key
+    enable: true
     enable_http2: true
     listen_port: 9443
     # ssl_trusted_certificate: /path/to/ca-cert # Specifies a file path with trusted CA certificates in the PEM format
                                                 # used to verify the certificate when APISIX needs to do SSL/TLS handshaking
                                                 # with external services (e.g. etcd)
-    # ssl_cert: /path/to/ssl_cert
-    # ssl_cert_key: /path/to/ssl_cert_key
+    ssl_cert: "cert/ssl_PLACE_HOLDER.crt"
```
Comment (membphis) [inline conf/config-default.yaml:110]: I think we can remove this option, the user does not need to care about it

File: conf/config-default.yaml [hunk 3]
```diff
@@ -101,15 +101,14 @@ apisix:
...
+    ssl_cert_key: "cert/ssl_PLACE_HOLDER.key"
```
Comment (membphis) [inline conf/config-default.yaml:111]: ditto

**Review by membphis — CHANGES_REQUESTED** [review]

**Comment by moonming** [comment]
nice

**Review by spacewander — APPROVED** [review]

**Review by moonming — APPROVED** [review]

File: conf/config-default.yaml (hunk 2 above)
Comment (tokers) [reply conf/config-default.yaml:110]: It's good to save this, since sometimes users may need to customize the default certs.

**Review by tokers — APPROVED** [review]

File: conf/config-default.yaml (hunk 2 above)
Comment (membphis) [reply conf/config-default.yaml:110]: this default cert is useless. and the APISIX never uses those certs.

Comment (Yiyiyimu) [reply conf/config-default.yaml]: Thanks! PTAL

**Review by membphis — APPROVED** [review]

**Pull request merged**